style.css

```diff
@@ -0,0 +1,200 @@
+@import url('https://fonts.googleapis.com/css?family=Nunito:400,400i,700');
+/* Eliminate bullet point, spread navigation on a row */
+
+body {
+    font-family: 'Nunito', sans-serif;
+}
+.nav {
+    display: grid;
+    grid-template-columns: 60% 30% 10%;
+    list-style-type: none;
+    font-weight: 700;
+   
+}
+.nav li:hover {
+    color: blue;
+}
+
+/* Create three columns to store each element of main-page */
+.intro {
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
```

Comment:
If you wanted to make the grid-template-rows and grid-template-columns a little more DRY here, you could try two things: 

1) the "grid-template" attribute can be used to determine both rows and columns. You write the measurement of your rows, followed by a /, followed by the measurements for your columns.

2) you can use "repeat" in the value to repeat multiple measurements. The syntax is "repeat(#, measurement)".

So, if you wanted, it could look like:

grid-template: repeat(6, 1fr) / repeat(7, 1fr)